src/ctia/entity/incident.clj

```diff
@@ -113,21 +113,31 @@
 (s/defschema IncidentStatusUpdate
   {:status IncidentStatus})
 
+(defn new-status? [status]
+   (some? (re-matches #"New(: .+)?" status)))
+
+(defn open-status? [status]
+   (some? (re-matches #"Open(: .+)?" status)))
+
+(defn closed-status? [status]
+  (some? (re-matches #"Closed(: .+)?" status)))
+
 (defn make-status-update
   [{:keys [status]}]
   (let [t (time/internal-now)
-        verb (case status
-               "New" nil
-               "Stalled" nil
-               "Hold" nil
-               ;; Note: GitHub syntax highlighting doesn't like lists with strings
-               "Containment Achieved" :remediated
-               "Restoration Achieved" :remediated
-               "Open" :opened
-               "Rejected" :rejected
-               "Closed" :closed
-               "Incident Reported" :reported
-               nil)]
+        verb (or (case status
+                   "New" nil
+                   "Stalled" nil
+                   "Hold" nil
+                   ;; Note: GitHub syntax highlighting doesn't like lists with strings
+                   "Containment Achieved" :remediated
+                   "Restoration Achieved" :remediated
+                   "Rejected" :rejected
+                   "Incident Reported" :reported
+                   nil)
+                 (cond
+                   (open-status? status) :opened
+                   (closed-status? status) :closed))]
```

Comment:
Are any additional changes needed here to account for new incident statuses? I wasn't sure. I only made the changes necessary to support interval computation.

Reply:
I think this looks good.

Reply:
that should be all in CTIA.
IROH changes may still be necessary, reach out to me offline please.